README.md

```diff
@@ -77,6 +77,15 @@ Octicons includes a stylesheet specifically for [Rails 4/Sprockets][sprockets].
     content_tag :span, '', :class => "octicon octicon-#{code.to_s.dasherize}"
   end
   ```
+  
+## Installing locally
+
+The easiest way to install Octicons locally is with Homebrew. Simply run the following commands:
```

Comment:
> The easiest way to install Octicons locally is with Homebrew.

I hesitate to agree with this, in that it requires you know what Homebrew is and that you have it installed. It might be better to say something like:

*It's easy to install octicons locally if you have [Homebrew](http://brew.sh/) installed.*